src/components/page-container/data/api.js

```diff
@@ -1,14 +1,8 @@
 import { getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
-const InstructorApiBaseUrl = `${getConfig().LMS_BASE_URL}/api/instructor/v1`;
 const courseHomeBaseUrl = `${getConfig().LMS_BASE_URL}/api/course_home/v1/course_metadata`;
 
-export default async function getInstructorData(courseId) {
```

Comment:
I searched the codebase and this must have been a leftover from an early part of our implementation. It wasn't being called anywhere.